tests/execution.rs

```diff
@@ -2356,7 +2374,7 @@ fn test_err_callx_unregistered() {
         mov64 r0, 0x2A
         exit",
         [],
-        TestContextObject::new(4),
+        TestContextObject::new(3),
```

Comment:
This would be consensus breaking, right?

Reply:
No, because the difference in program counter and CU only occurs inside the error branch we will enable in SBFv2:

https://github.com/solana-labs/rbpf/blob/8d36530b7071060e2837ebb26f25590db6816048/src/interpreter.rs#L522-L523

Reply:
Ok, then can you have it test both SBPF versions? One `Ok` and one `Err` case.

Reply:
I added a test for V1 too. It is not an OK case, but a `CallOutsideTextSegment` error.